packages/back-end/src/app.ts

```diff
@@ -651,7 +651,11 @@ app.put(
 );
 app.get(
   "/experiments/launch-checklist",
-  experimentLaunchChecklistController.getExperimentCheckListByOrg,
+  experimentLaunchChecklistController.getExperimentCheckListForOrg,
+);
+app.get(
+  "/experiment/:id/launch-checklist",
+  experimentLaunchChecklistController.getExperimentCheckListForExperiment,
 );
```

Comment:
Now that a checklist's items can differ from experiment to experiment (depending on what project an experiment is in), I created a new endpoint to fetch the pre-launch checklist for a specific experiment. 

This checklist is separate from the general endpoint to fetch the checklist for the `/settings` page that doesn't take into consideration a particular experiment.